V1Documentation.md

```diff
@@ -453,17 +387,6 @@ Raygun.init('apikey', { disableAnonymousUserTracking: true });
 You can provide additional information about the currently logged in user to Raygun by calling:
 
 ```javascript
-// V2
-rg4js('setUser', {
-  identifier: 'user_email_address@localhost.local',
-  isAnonymous: false,
-  email: 'emailaddress@localhost.local',
-  firstName: 'Foo',
-  fullName: 'Foo Bar',
-  uuid: 'BAE62917-ACE8-ab3D-9287-B6A33B8E8C55'
-});
-
-// V1
 Raygun.setUser('user_email_address@localhost.local', false, 'user_email_address@localhost.local', 'Foo Bar', 'Foo', 'BAE62917-ACE8-ab3D-9287-B6A33B8E8C55');
```

Comment:
If we can format each property on newlines like the V2 example with comments after each line that'd help show what each property is